### ISSUE TITLE
NVDA+Firefox does not read contents of an element referenced with aria-describedby if the element has role="alert"

### ISSUE THREAD

**Issue by Wildebrew** [nvaccess/nvda#8929]
### Steps to reproduce:

1. With NVDA and Firefox running, go to [my Code pen](https://codepen.io/Wildebrew/pen/QJNQXB)
2. Use the tab key to move to the First name field. The error message should be announced.
3. Tab to the "lastname" field. The error message is not announced.

### Actual behavior:
Error message is not announced for the last name field.

### Expected behavior:
Error message should be announced for the last name.

### System configuration:
NVDA 2018.1 with Firefox 60 (verified with Firefox 61).

#### NVDA version:
2018.1 (also tested with 2018.3)

**Comment by Adriani90** [comment]
cc: @jcsteh

**Comment by jcsteh** [comment]
This is a Firefox bug. Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1505974

**Comment by jcsteh** [comment]
Fixed for Firefox 75 (currently available for testing in Firefox Nightly).
